src/entities/message.js

```diff
@@ -0,0 +1,33 @@
+const BaseEntity = require('./base')
+
+class Message extends BaseEntity {
+    set text(text) {
+        if (!text) throw new Error('Message is required!')
+        this.attr.text = text.trim()
+    }
+
+    get text() {
+        return this.attr.text
+    }
+
+    get messageRoom() {
+        return this.attr.roomId
+    }
+
+    get messageCreatedDate() {
+        return this.attr.createdAt
+    }
+
+    toJSON() {
+        return {
```

Comment:
most of attributes of returned json will be undefined.
it returns roomId: undefined, senderId: undefined, ...

this method declaration can be removed. in this case will be used parent implementation

Reply:
Removed.